blog-posts/staticIpLambda/staticIpLambda.md

```diff
@@ -0,0 +1,193 @@
+---
+published: false
+title: 'Deploying a Lambda with a static IP has never been so simple 🍰'
+cover_image:
+description: 'Learn how to deploy a lambda with a static IP (for whitelisting concerns) and perform NodeJS SFTP operations using this lambda.'
+tags: serverless, sftp, aws, lambda
+series:
+canonical_url:
+---
+
+## TL;DR <br>
+🧠 Learn how to deploy a **lambda with a static IP** (for whitelisting concerns)
+⚡ Perform **NodeJS SFTP operations** using this lambda. <br>
+
+---
+
+Your Serverless application might need to connect to a partnet server that requires IP whitelisting.
+We will use the [Serverless Framework](https://www.serverless.com/) to deploy our lambda function and the **Typescript AWS CDK** to create the infrastructure needed to give our lambda a static IP 🔒. <br>
+Moreover, we will use the NodeJS [ssh2-promise](https://www.npmjs.com/package/ssh2-promise) package to perform SFTP operations in our lambda function. 💽
+
+This is what we will build 👷:
+
+![Static Lambda IP Architecture](./assets/archi.png 'Logo Kumo')
+
+Let's get to work 💪🔨!
+
+# 1️⃣ Create a VPC hosted Lambda function with a static IP
+
+## ☁️ VPC, Nat Gateway and Elastic IP
+
+All the resources will be deployed using the **Serverless Framework** and the **Typescript AWS CDK**. If you are not familiar with it, I recommend you check out this [Serverless Framework x CDK blog post](https://dev.to/kumo/serverless-framework-aws-cdk-1dnf) to learn how to conciliate the two. 🤝
```

Comment:
Remonte ca un peu, genre vers la ligne 18. Ca arrive un peu tard je trouve.

tu as aussi cet article plus récent: https://dev.to/kumo/the-ultimate-serverless-devx-serverless-framework-aws-cdk-and-typescript-3694